server/memcached/src/main/scala/org/infinispan/server/memcached/configuration/MemcachedServerConfiguration.java

```diff
@@ -15,8 +15,8 @@
 @BuiltBy(MemcachedServerConfigurationBuilder.class)
 public class MemcachedServerConfiguration extends ProtocolServerConfiguration {
 
-   MemcachedServerConfiguration(String defaultCacheName, String name, String host, int port, int idleTimeout, int recvBufSize, int sendBufSize, SslConfiguration ssl, boolean tcpNoDelay, int workerThreads, Set<String> ignoredCaches) {
-      super(defaultCacheName, name, host, port, idleTimeout, recvBufSize, sendBufSize, ssl, tcpNoDelay, workerThreads, ignoredCaches);
+   MemcachedServerConfiguration(String defaultCacheName, String name, String host, int port, int idleTimeout, int recvBufSize, int sendBufSize, SslConfiguration ssl, boolean tcpNoDelay, int workerThreads, Set<String> ignoredCaches, boolean startTransport) {
```

Comment:
Yes, but Memcached server still extends the `ProtocolServer` so we need to adjust the code.